tensorflow/core/profiler/internal/gpu/cupti_collector.cc

```diff
@@ -487,6 +487,11 @@ class PerDeviceCollector {
 
   void GetDeviceCapabilities(int32 device_ordinal,
                              XPlaneBuilder* device_plane) {
+    std::string device_manfacturer = "Nvidia";
```

Comment:
absl::string_view device_manufacturer = "Nvidia";

Fix typo (manfacturer).
absl::string_view should work here to avoid an unnecessary string copy.